src/System.Reflection.DispatchProxy/src/System/Reflection/DispatchProxyGenerator.cs

```diff
@@ -426,13 +426,68 @@ internal void AddInterfaceImpl(Type iface)
                 _assembly.EnsureTypeIsVisible(iface);
 
                 _tb.AddInterfaceImplementation(iface);
+
+                // AccessorMethods -> Metadata mappings.
+                var propertyMap = new Dictionary<MethodInfo, PropertyAccessorInfo>();
+                var eventMap = new Dictionary<MethodInfo, EventAccessorInfo>();
```

Comment:
Nit: I'd prefer the declaration of "eventMap" to be grouped with the "GetRuntimeEvents()" loop below rather than appearing up here.